### DISCUSSION TITLE
Biometric login feature request

### DISCUSSION THREAD

**Discussion by App-Monkeys** [inspireui/support#5717] [New Feature/Ideas Requests] [answered]
Requesting that biometric/ facial recognition as login be added.

If anyone knows how to do this currently it would be of great help as i've tried and failed!

**Comment by HanielQC** [comment, marked as answer]
This feature is available on FluxStore app 3.8.0 - Don't miss the 30% sale this March 2023 💥🥳
https://app.inspireui.com/woocommerce/changelog
